manuskript.spec

```diff
@@ -38,3 +49,11 @@ coll = COLLECT(exe,
                strip=False,
                upx=True,
                name='manuskript')
+
+# also build an app bundle on macOS
+if platform == 'mac':
```

Comment:
None of the existing build scripts in this file do any platform detection. Better yet, the different actions taken appear to be platform agnostic despite targeting the creation of a Windows Exe, among other things. Can the platform check be removed entirely, allowing for the bundle be generated on other machines? Or is there something preventing this?

I do not believe any of the maintainers have a Mac, so if building the app bundle requires a mac, it would be less than ideal when they want to release a new version.

Reply:
Correct, none of the current maintainers have a Mac.

@dschanoeh does this PR change the file name of the macOS X package?  
If so then we will need to handle the change in Travis CI.

In addition to other things, Travis CI is used to build the macOS X package which is controlled by the [.travis.yml](https://github.com/olivierkes/manuskript/blob/develop/.travis.yml) file.  Note that the .travis.yml file uploads the `dist/manuskript-osx-develop.zip` file to our web server because all files in Travis CI disappear after Travis CI has run.

The original Manuskript author @olivierkes also set up Semaphore CI, which I believe uploads to [Github Releases](https://github.com/olivierkes/manuskript/releases) but unfortunately we do not have access to change how that works.  All we can do there is make a request to @olivierkes, or create work-arounds.

Reply:
As far as I understand, the Mac package gets built in a macOS environment by Travis CI, right? In the end, there will be Mac specific binaries (e.g. the QT library) in the bundle. If we remove the platform check here, you might be able to build an App Bundle (which really is just a folder with defined structure) on e.g. Windows but I'm assuming the resulting bundle wouldn't run on the Mac.
I understand that it would be cleaner to not have a check but I think you'd always be building an additional folder on Windows/Linux that wouldn't be useful to anybody.

Regarding the capital "M" in the bundle name: It is common on the Mac to use title case for app file names. For example Wireshark's app is called 'Wireshark.app' even though the command line binary is 'wireshark'. I've seen capital 'Manuskript' used everywhere on the home page and therefore thought that would be consistent. Let me know if that needs to change.

Reply:
Correct.  The mac package gets built in a macOS environment by the Travis CI.  Note that the Travis CI also runs for GNU/Linux so code in the `.travis.yml` file must take this into account.

I am okay either with or without the capital "M" in "Manuskript.app".  The key point is that a change in filename must also be reflected in the code that runs in Travis CI.  We also have to make corresponding changes to the [Manuskript Download](https://www.theologeek.ch/manuskript/download/) page.